packages/text-bitmap/src/BitmapText.ts

```diff
@@ -473,7 +488,7 @@ export class BitmapText extends Container
         for (let i = 0; i < lenChars; i++)
         {
             const char = chars[i];
-            let offset = char.position.x + lineAlignOffsets[char.line];
+            let offset = char.position.x + (lineAlignOffsets[char.line] * char.prevSpaces);
```

Comment:
This line is now quite confusing (although I understand in the context of this PR, "adding line justify"). The problem is that `prevSpaces` is 1 for all other alignments, and that is not at all obvious. There may be actually character spaces before `char` on that line, but we are ignoring them for non-justify alignments

I would suggest keeping `prevSpaces` the actual number of spaces before the character. Then, add an ternary operator to multiply it only when the alignment is justify.

```suggestion
            let offset = char.position.x + (lineAlignOffsets[char.line] * (this._align === 'justify' ? char.prevSpaces : 1));
```